psp-examples/swap/tests/swaps.ts

```diff
@@ -69,6 +69,10 @@ const createTestUser = async (
 describe("Test swaps", () => {
   process.env.ANCHOR_PROVIDER_URL = RPC_URL;
   process.env.ANCHOR_WALLET = process.env.HOME + "/.config/solana/id.json";
+  // Enable atomic transactions if they weren't explicitly disabled.
+  if (process.env.LIGHT_PROTOCOL_ATOMIC_TRANSACTIONS !== "false") {
```

Comment:
We do it here:

https://github.com/Lightprotocol/light-protocol/blob/c583553547a026d1875a7b435a73d5b32b44ea96/.github/workflows/psp-examples-tests.yml#L36-L38

The point of the diff you are commenting here is to enable atomic TX by default and disable it optionally (the opposite of what we do for the CLI and SDK in general). Why? Because disabling them by default would mean having to pass `LIGHT_PROTOCOL_ATOMIC_TRANSACTION=true` every time you want to run tests locally - easy to forget.